velox/docs/configs.rst

```diff
@@ -108,3 +108,16 @@ can be used to prevent a query from using too much io and cpu resources.
     * **Default value:** ``0``
 
 The maximum allowed spill file size. Zero means unlimited.
+
+``min-spill-run-siz``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``0``
```

Comment:
Should the default be something other than zero?

Reply:
Yeah, I changed it to 256MB for now.